tests/aggregation/tests.py

```diff
@@ -988,6 +988,30 @@ def test_annotated_aggregate_over_annotated_aggregate(self):
         with six.assertRaisesRegex(self, FieldError, "Cannot compute Sum\('id__max'\): 'id__max' is an aggregate"):
             Book.objects.annotate(Max('id')).annotate(Sum('id__max'))
 
+        class MyMax(Max):
+            def as_sql(self, compiler, connection):
+                self.set_source_expressions(
+                    self.get_source_expressions()[0:1])
+                return super(MyMax, self).as_sql(compiler, connection)
+
+        with six.assertRaisesRegex(self, FieldError, "Cannot compute Max\(...'id__max'...\): 'id__max' is an aggregate"):
```

Comment:
I'm not very happy with this error message (the ellipsis at the beginning and end). Should we not worry about differentiation between one and multi-arg aggregates that ref other aggregates? I'd rather not build out the entire string of args for some reason, even though it'd be more accurate.